_posts/2021-02-24-without-data-we-re-blind.md

```diff
@@ -0,0 +1,107 @@
+
+15% of unplanned work is the average for 2020. During summer it was still around 18%, and now it is down to 11%. Given that the situation was the same, I sat down once more for another round of analysis around August 2020.
+
+Why that amount of unplanned work? Looking in-depth at my calendar week by week, it was clear that activities were too fragmented. If there is a recurring 1-hour sync call at 10 AM European time, the morning became a context switching mess.
+
+The other problem was that a 100% increase of "day to day tasks" led to an even worse situation. We work in small groups, and we tend to pair a lot; I was joining too many activities, which means joining too many groups, which resulted in even more fragmentation. Each day I was working with three or four different groups, on top of all the other things.
+
+I had two options: give up on a few things or find a solution to reduce context switching hell. I did not want to give up on anything, and to some extent giving up on some things giving up was not an option at all.
+
+## I did both
+
+The first thing was to rearrange as much as I could my calendar to group similar things. For example, during recurring product management syncs, we capture action items, and at the end of the sync, we assign action items to each participant. It's easy to view those action items as tasks on a to-do list, in which case the problem for me is that they get a very low priority (remember that I'm a procrastinator). Instead, I scheduled a recurring 45-minute slot ten minutes after every sync to get all the assigned action items done.
+
+> Tip: always leave a buffer between scheduled calendar events. Things can exceed their planned time, or we need a break, coffee, tee, or a snack, or the restroom.
+
+I didn't stop myself from defining "action items" slots on the calendar. I spent a significant amount of time rearranging my weekly schedule, which was enough to alleviate some of the problems. Again, with the goal of grouping similar activities.
+
+The second thing was to serialize more and parallelize less. How? Join fewer groups, but more importantly, be more conscious about what you join. Theoretically, zero context switching is possible if we were doing one thing. In my case, and probably this is true for everyone, it's impossible, and I'm okay with it. Instead of trying to be on top of everything every day, I moved, where possible, to a two to three-hour-long slots calendar. When working on group assigned tasks, I try to allocate to each task two to three hours. Those are long enough to get a lot done, short enough for my attention span, and compensate for the context switching effect well. At the beginning of the slot, it takes me about 15 minutes to answer the question, "where did I left?" And then have a significant amount of time at full speed. At the end of the slot, the ten or fifteen minutes buffer works like a decompression thing to prepare me for the next thing. 
```

Comment:
```suggestion
The second thing was to serialize more and parallelize less. How? Join fewer groups, but more importantly, be more conscious about what you join. Theoretically, zero context switching is possible if we were doing one thing. In my case, and probably this is true for everyone, it's impossible, and I'm okay with it. Instead of trying to be on top of everything every day, I moved, where possible, to a two to three-hour-long slots calendar. When working on group assigned tasks, I try to allocate to each task two to three hours. Those are long enough to get a lot done, short enough for my attention span, and compensate for the context switching effect well. At the beginning of the slot, it takes me about 15 minutes to answer the question, "where did I leave off?" And then have a significant amount of time at full speed. At the end of the slot, the ten or fifteen minutes buffer works like a decompression time to prepare me for the next thing. 
```